Sources/Foundation/UserDefaults.swift

```diff
@@ -319,16 +320,16 @@ open class UserDefaults: NSObject {
         return names
     }
     
-    open func volatileDomain(forName domainName: String) -> [String : Any] {
+    open func volatileDomain(forName domainName: String) -> [String: Any] {
```

Comment:
I would suggest not do such changes since `[Key : Value]` is more commonly used to express a `Dictionary` type. Its symmetrical look is much neater than the unsymmetrical `[Key: Value]`.